protocols/imap/src/test/java/org/apache/james/imap/encode/AnnotationResponseEncoderTest.java

```diff
@@ -0,0 +1,88 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.imap.encode;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.IOException;
+
+import com.google.common.collect.ImmutableList;
+import org.apache.james.imap.encode.base.ByteImapResponseWriter;
+import org.apache.james.imap.encode.base.EndImapEncoder;
+import org.apache.james.imap.encode.base.ImapResponseComposerImpl;
+import org.apache.james.imap.message.response.AnnotationResponse;
+import org.apache.james.mailbox.model.MailboxAnnotation;
+import org.junit.Test;
+
+import com.google.common.collect.Lists;
+
+public class AnnotationResponseEncoderTest {
+
+    private static final MailboxAnnotation PRIVATE_ANNOTATION = MailboxAnnotation.newInstance("/private/comment", "My own comment");
+    private static final MailboxAnnotation SHARED_ANNOTATION = MailboxAnnotation.newInstance("/shared/comment", "Shared comment");
+
+    @Test
+    public void annotationMessageResponseShouldBeWellFormattedWithoutReturnAnnotation() throws Exception {
+        AnnotationResponse response = new AnnotationResponse("INBOX", ImmutableList.<MailboxAnnotation>of());
+        ByteImapResponseWriter byteImapResponseWriter = new ByteImapResponseWriter();
+        ImapResponseComposer composer = new ImapResponseComposerImpl(byteImapResponseWriter, 1024);
+        AnnotationResponseEncoder encoder = new AnnotationResponseEncoder(new EndImapEncoder());
+        encoder.encode(response, composer, null);
+        String responseString = byteImapResponseWriter.getString();
+
+        assertEquals("* METADATA \"INBOX\"\r\n", responseString);
```

Comment:
Use assertThat every where.

The problem with assertEquals is that it is not obvious if it is

assertEquals(actual, expected) or assertEquals(expected, actual)